counterparty-core/counterpartycore/lib/p2sh.py

```diff
@@ -0,0 +1,179 @@
+"""
```

Comment:
So I think we're going to be removing P2SH encoding soon-ish (once we add support for encoding in witness data). In the meantime, we can re-enable this? Will reduce fees for some users.